devices/Sim7080/ConnectionStatus.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace IoT.Device.Sim7080
```

Comment:
once, you'll add StyleCop, you'll see that you need a header, it's mandatory in all the CS file.
And that you need to have the intellisense comments on all public elements.

Reply:
Added the headers and summaries to multiple files.